### PR TITLE
Banner link update and tiny copy edit

### PR DESCRIPTION
**Pull request by faihegberg**
Addresses two cards:
- https://trello.com/c/RLWaFHWH/599-copy-update-hsa-names
- https://trello.com/c/tUtKiung/597-update-the-target-of-the-learn-more-link-in-the-banner

### CODE REVIEW

**Comment by faihegberg** [comment]
[Copy update - HSA names](https://trello.com/c/RLWaFHWH/599-copy-update-hsa-names)

**Comment by faihegberg** [comment]
[Update the target of the Learn more link in the banner](https://trello.com/c/tUtKiung/597-update-the-target-of-the-learn-more-link-in-the-banner)

File: src/components/Banner/Can82Banner.tsx [hunk 1]
```diff
@@ -4,15 +4,29 @@ import {
...
-import { Link } from 'react-router-dom';
+import ExternalLink from 'components/ExternalLink';
+import { EventAction, EventCategory, trackEvent } from 'components/Analytics';
 
-//TODO(8.2) - update with correct learn more URL
+const trackClick = () => {
+  trackEvent(
+    EventCategory.HOMEPAGE_BANNER,
+    EventAction.CLICK_LINK,
+    'Banner Learn More Link',
```
Comment (pnavarrc) [inline src/components/Banner/Can82Banner.tsx:14]: The category and action already tell you that the user is clicking a link in the banner - consider changing the label to something more descriptive `Learn Community Levels` or something like that.

**Review by pnavarrc — APPROVED** [review]
Looks good, except for the tracking label, feel free to merge once updated!

**Pull request merged**